File: app/src/main/java/dev/bee/kanjianki/StudyTokenFactory.java
package dev.bee.kanjianki;

import dev.bee.kanjianki.core.StudyTokenPolicy;

final class StudyTokenFactory {
    private StudyTokenFactory() {
    }

    static String studyItem(String kanji, String activeToken) {
        return StudyTokenPolicy.studyItem(kanji, activeToken);
    }
}

File: app/src/main/kotlin/dev/bee/kanjianki/StudyTokenFactory.kt
package dev.bee.kanjianki

import dev.bee.kanjianki.core.StudyTokenPolicy

internal object StudyTokenFactory {
    @JvmStatic
    fun studyItem(kanji: String?, activeToken: String?): String {
        return StudyTokenPolicy.studyItem(kanji, activeToken)
    }
}
